sources/models/src/modeled_types/credentialproviders.rs

```diff
@@ -51,23 +51,14 @@ mod test_cacheduration_value {
 
     #[test]
     fn valid_cacheduration_value() {
-        for ok in &[
-            "12h",
-            "120m",
-            "2h15m"
-        ] {
+        for ok in &["12h", "120m", "2h15m"] {
```

Comment:
This file was created in a different commit, and it was formatted here. You should have the formatted version in the commit that created the file.